cajita/unit_test/tstHalo.hpp

```diff
@@ -54,6 +54,7 @@ int haloPad( Edge<D>, int d )
 template <class Array>
 void checkGather( const int halo_width, const Array& array )
 {
+    int pass_gather_test = 1;
```

Comment:
Were these changes part of trying to address the failing tests?

Reply:
Yes, just limiting the prints because it was hanging

Reply:
Do you want a `fixme` for each?

Reply:
yes let's add this. Seeing as it was distribution dependent we will really want to dig.